src/api/todos.ts

```diff
@@ -0,0 +1,30 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const createTodo = async (title: string, userId: number) => {
+  const response = await client.post<Todo>('/todos', {
+    title,
+    userId,
+    completed: false,
+  });
+
+  return response;
```

Comment:
These functions should return a promise
```suggestion
  return client.post<Todo>('/todos', {
    title,
    userId,
    completed: false,
  });

```